pkg/invoke/invoker.go

```diff
@@ -334,6 +334,9 @@ func (i *Invoker) Agent(ctx context.Context, c kclient.WithWatch, agent *v1.Agen
 		credContextIDs = append(credContextIDs, agent.Name)
 	}
 	credContextIDs = append(credContextIDs, agent.Namespace)
+	if len(agent.Spec.AdditionalCredentialContexts) != 0 {
+		credContextIDs = append(credContextIDs, agent.Spec.AdditionalCredentialContexts...)
+	}
```

Comment:
nit: this does the right thing, even for `nil` slices.
```suggestion
	credContextIDs = append(credContextIDs, agent.Spec.AdditionalCredentialContexts...)
```